src/pages/Mypage/Order/Order.js

```diff
@@ -1,7 +1,251 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ReactDaumPost from 'react-daumpost-hook';
+import { API } from '../../../config';
+import { Card, CardMedia, TextField, Button } from '@mui/material';
+import * as S from './Order.style';
 
 const Order = () => {
-  return <div>Order</div>;
+  const [orderData, setOrderData] = useState({});
+  const [loading, setLoding] = useState(true);
+  const [addressData, setAddressData] = useState({
+    bidId: Number.MAX_SAFE_INTEGER,
+    phoneNumber: '',
+    street: '',
+    address: '',
+    zipcode: '',
+    price: Number.MAX_SAFE_INTEGER,
+  });
+  const { street, address, zipcode, phoneNumber } = addressData;
+  const {
+    imageUrl,
+    userName,
+    userEmail,
+    itemName,
+    authorName,
+    bidPrice,
+    shippingFee,
+    totalPrice,
+  } = orderData;
+  const orderPriceList = [
+    {
+      id: 1,
+      name: '낙찰금액',
+      낙찰금액: bidPrice,
+    },
+    {
+      id: 2,
+      name: '배송비',
+      배송비: shippingFee,
+    },
+    {
+      id: 3,
+      name: '부가세',
+      부가세: '0원',
+    },
+  ];
+
+  const handleAddressDatailChange = e => {
+    const { name, value } = e.target;
+
+    if (name === 'phoneNumber') {
+      const phoneNumber = value.replace(/-/g, '');
+      const regex = /^(\d{2,3})(\d{3,4})(\d{4})$/;
```

Comment:
```suggestion
      // 최소 9자리부터 최대 11자리까지 입력가능한 숫자 
      // ex)000 0000 0000 
      const regex = /^(\d{2,3})(\d{3,4})(\d{4})$/;
```
이렇게 주석으로 설명해주면, 표현식 해석하기 수월 할 것같습니다.